src/utils/hooks/api/get-vault-overview.ts

```diff
@@ -81,12 +83,27 @@ const getVaultOverview = async (
     })
   });
 
+  const redeems = await fetch(HYDRA_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      query: redeemCountQuery(
+        `vault: {accountId_eq: "${accountId.toString()}", collateralToken_eq: ${tokenIdLiteral}}, status_eq: Pending`
+      )
+    })
+  });
+
   const issuesCount = await issues.json();
+  const redeemsCount = await redeems.json();
+
+  const pendingRequests = issuesCount.data.issuesConnection.totalCount + redeemsCount.data.redeemsConnection.totalCount;
```

Comment:
What we have in the UI is 'Pending requests', which is intended to be issues and redeems—if we want to change that for any reason we can though :)

Reply:
My suggestion 

> Good solution :) However it's diverging from the ticket a little, I think it would be good to discuss whether to display both pending issue and redeem requests or redeem requests only. @tomjeatt @sander2 cc @alexbleau

My suggestion here would be that we combine them into a single value (as in the PR) because that's what in the UI, and in that respect the number is wrong! We can easily change the UI if we want to show only redeems (or add an extra line and show pending redeems/issues separately) but we'd want to have a chat with Jay about that.